enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -79,23 +82,30 @@ export function useLoadQuestion({
   // Avoid re-running the query if the parameters haven't changed.
   const sqlParameterKey = getParameterDependencyKey(initialSqlParameters);
 
-  const [loadQuestionState, loadQuestion] = useAsyncFn(async () => {
-    const state = await dispatch(
-      runQuestionOnLoadSdk({
+  const [loadQuestionState, loadAndQueryQuestion] = useAsyncFn(async () => {
```

Comment:
This is only a rename, with no base functionalities change. But there's a change in the function itself, I'll explain more later.